wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -103,7 +103,7 @@ const Lobby = ({location}) => {
     return (
         <Layout>
             <div className="outerContainer">
-                <div className="container">
+                <div className= "lobbyContainer">
```

Comment:
I like how the CSS changes can be applied to both of the major components on this page simulataneously. Good thinking, this is a good application of the DRY principle!